src/sentry/api/bases/organization_events.py

```diff
@@ -102,25 +103,36 @@ def get_snuba_query_args_legacy(self, request, organization):
             )
         return snuba_args
 
-    def next_event_id(self, request, organization, snuba_args, event):
+    def next_event_id(self, snuba_args, event):
```

Comment:
This and `prev_event_id` seems pretty similar, can we consolidate?

Reply:
Yep. I have some more clean up to do here as well.